function-reference/xml2json.md

```diff
@@ -0,0 +1,19 @@
+---
+description: Convert xml string to json string
```

Comment:
@hetunandu incorrect description

Reply:
@hetunandu this is still not resolved. We're not returning a json string but an object right?